api/envoy/config/listener/v3/quic_config.proto

```diff
@@ -24,11 +24,19 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // [#protodoc-title: QUIC listener config]
 
 // Configuration specific to the UDP QUIC listener.
-// [#next-free-field: 12]
+// [#next-free-field: 13]
 message QuicProtocolOptions {
   option (udpa.annotations.versioning).previous_message_type =
       "envoy.api.v2.listener.QuicProtocolOptions";
 
+  // Configuration for UDP cmsg to pass to listener filters via QuicReceivedPacket.
```

Comment:
What is cmsg?
Are there links that can be added here that provide more info?

Reply:
See https://man7.org/linux/man-pages/man3/cmsg.3.html, basically socket control info. I've added a link in the proto.